src/graph/ExecutionPlan.cpp

```diff
@@ -92,6 +102,29 @@ void ExecutionPlan::onError(Status status) {
     auto latency = rctx->duration().elapsedInUSec();
     stats::Stats::addStatsValue(allStats_.get(), false, latency);
     rctx->resp().set_latency_in_us(latency);
+
+    auto errorCodeIt = cpp2::_ErrorCode_VALUES_TO_NAMES.find(rctx->resp().get_error_code());
+    LOG(ERROR) << "Execute failed! errcode: "
+        << (errorCodeIt != cpp2::_ErrorCode_VALUES_TO_NAMES.end()
+                ? errorCodeIt->second
+                : std::to_string(int32_t(rctx->resp().get_error_code())))
+        << ", cost: " << latency
+        << "us, space: " << (rctx->session()->spaceName())
+        << ", query: " << (rctx->query().c_str())
```

Comment:
I think you could remove the `.c_str` call.

Reply:
Thanks, I have modified this.